documentation/swagger-checklist.md

```diff
@@ -1,6 +1,6 @@
 # Swagger Checklist for Microsoft Azure #
 
-*Version 0.1, 11-28-2016*
+*Version 0.1, 12-12-2016*
```

Comment:
We should add some information that we are working on a tool, and these MUST/SHOULD haves will move to a compiler/analyzer kind of tool to support swagger authoring.
This is a long list and it may be just a start (will be growing, as we will be finding more issues with swagger authoring).
Also as swagger authors will be putting a lot of operations/definitions in the swagger, it is be better to categorize MUST/SHOULD have further so that it is easy to find what needs to be done in what scenarios.